integration-cli/docker_cli_authz_plugin_v2_test.go

```diff
@@ -36,8 +36,10 @@ func (s *DockerAuthzV2Suite) SetUpTest(c *check.C) {
 }
 
 func (s *DockerAuthzV2Suite) TearDownTest(c *check.C) {
-	s.d.Stop()
-	s.ds.TearDownTest(c)
+	if s.d != nil {
+		s.d.Stop()
+		s.ds.TearDownTest(c)
```

Comment:
the ones in check_test.go should be changed to have the TearDownTest inside the nil daemon check too then, right?